app/src/main/java/net/tensory/rxjavatalk/house/HouseFragment.java

```diff
@@ -21,15 +21,18 @@
 import net.tensory.rxjavatalk.injection.AppComponent;
 import net.tensory.rxjavatalk.models.House;
 
+import java.util.Locale;
+
 import io.reactivex.android.schedulers.AndroidSchedulers;
-import io.reactivex.disposables.Disposable;
+import io.reactivex.disposables.CompositeDisposable;
 
 public class HouseFragment extends Fragment {
 
     public static final String ARG_HOUSE = "ARG_HOUSE";
 
     private HousePresenter presenter;
-    private Disposable disposable;
+
+    private CompositeDisposable compositeDisposable = new CompositeDisposable();
```

Comment:
I think we need to create a new one for each onStart, because I believe that CompositeDisposable can only be disposed once

Reply:
Good catch, thanks for helping me understand that. I remember the distinction between `clear` and `dispose`.